### ISSUE TITLE
Select a model - screen reader issues

### ISSUE THREAD

**Issue by iamjolly** [GSA-TTS/10x-ai-sandbox#414]
## Environment

Localhost on `main` branch, Safari and Chrome browsers, VoiceOver 

## Expected Behavior:

When opening the ModelSelector component, the model names and currently selected model should be announced to screen reader users.

## Actual Behavior:

Model names are not announced as shown visibly. Instead, they are all announced as "model-item" and do not indicate the selected state audibly as shown by the checkmark on screen for the selected model.

Additionally, there are GSA logos next to each model name _(EDIT: These seem to be GSA-only logos in local environments)_. ~~These are likely not needed and likely are misleading for the models' actual logos.~~ They also have generic `alt` text which are not accurate or needed.

## Description

**Bug Summary:**
`aria-label`s are being misused in the ModelSelector component. Removing the `aria-label`s will allow the model text to be read as the accessible name for each `button` element.

## Logs and Screenshots

**Screen Recording (if applicable):**

https://github.com/user-attachments/assets/d2f6b703-0ccc-47de-aa81-81620ea78246

**Comment by iamjolly** [comment]
My latest commits now have this _mostly_ working as intended. Here's a screen recording (there's no audio other than what the screereader captions show) showing VoiceOver on Safari with the model names being read out and the check mark showing which model is selected audibly to match the visible checkmark.

https://github.com/user-attachments/assets/e027cae5-9151-4296-9642-7aa3faecb50a

**Comment by iamjolly** [comment]
Dev and Prod showing correct model logos, so the logos question is moot.

![Prod showing correct logos for the available models](https://github.com/user-attachments/assets/1657f16d-fc58-4011-870a-a9749ec6829d)

**Comment by vickimcfadden** [comment]
Moving to done as completed.
